src/transit_equity/geospatial/hex_to_shapely.py

```diff
@@ -0,0 +1,21 @@
+import geopandas as gpd
+from shapely import wkb
+import binascii
+
+def load_wkb(hex_string):
+    """Function to decode and load WKB binary location
+    into Shapely geometry object to enable plotting
+    with geoPandas.
+    
+    Parameters
+    ----------
+    hex_string : object
+        WKB binary object
+
+    Returns
+    -------
+    object
+        Shapely geometry object.
+
+    """
+    return wkb.loads(binascii.unhexlify(hex_string))
```

Comment:
Alternative code that does not need as many dependencies:
`wkb.loads(bytes.fromhex(row['transaction_location']))`

Reply:
Not necessary to change it tho. Just a suggestion in case we want to avoid extra dependencies.